.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "initializer_list": "cpp"
+    }
+}
```

Comment:
Looks good, could we just remove this file and add `.vscode` to the .gitignore?

Reply:
Thanks for pointing that out! Just fixed it!